resources/assets/js/components/PostList.vue

```diff
@@ -1,12 +1,13 @@
 <template>
     <tbody>
         <tr>
-            <td class="text-right font-bold pr-4 pt-4">{{ source.name }}</td>
-
-            <td></td>
-
-            <th v-for="target in targets" class="pr-4 text-sm">
-                <a :href="target.url" target="_blank">{{ target.name }}</a>
+            <th class="">{{ source.name }}</th>
+            <th :key="target.id" v-for="target in targets" class="pr-4 text-sm" colspan="2">
+                <a
+                :href="target.url"
+                target="_blank"
+                :title="target.name"
+                >{{ target.name }}</a>
```

Comment:
Did we ever decide how to format tags with multiple attributes like this? [Earlier on in the PR](https://github.com/tightenco/postit/pull/2/files#diff-13f43477847ecb5a8a698a16f3c9eaf0R4) we're doing it like so
```
    <a
         :href="post.guid"
         :title="this.post.title"
         class="no-underline hover:underline font-normal text-grey-darkest"
     >
```
and we should probably be consistent one way or the other